packages/appkit/src/adapters/ChainAdapterConnector.ts

```diff
@@ -0,0 +1,6 @@
+import type { CaipNetwork } from '@reown/appkit-common'
+import type { Connector } from '@reown/appkit-core'
+
+export interface ChainAdapterConnector extends Connector {
+  chains: CaipNetwork[]

```

Comment:
We might need to use `AppKitNetwork` instead of `CaipNetwork`. `AppKitNetwork` includes both Viem network and `CaipNetwork`. Both `createAppKit` and `WagmiAdapter` should be able to handle Viem networks as well.

We are extending networks while initializing the AppKit